testdata/acquisition/covid_hosp/state_timeseries/metadata.csv

```diff
@@ -0,0 +1,9 @@
+Update Date,Days Since Update,User,Rows,Row Change,Columns,Column Change,Metadata Published,Metadata Updates,Column Level Metadata,Column Level Metadata Updates,Archive Link
```

Comment:
This also appears to be for facility and not state

Reply:
replaced with state timeseries metadata